conf/uppmax.config

```diff
@@ -77,6 +91,15 @@ if (hostname.startsWith("i")) {
   params.config_profile_description = 'UPPMAX (Irma) cluster profile provided by nf-core/configs.'
 }
 
+// Cluster: Miarka
+if (hostname.startsWith("m")) {
+  params.max_memory = 357.GB
+  // but what about the 96-core node?
```

Comment:
I can tell you what - Nextflow will never request it 😅

Reply:
👌